ShopperInsights/src/main/java/com/braintreepayments/api/shopperinsights/v2/ShopperInsightsClientV2.kt

```diff
@@ -123,4 +125,26 @@ class ShopperInsightsClientV2 internal constructor(
     fun isVenmoAppInstalled(context: Context): Boolean {
         return deviceInspector.isVenmoInstalled(context)
     }
+    /**
+     * Updates the customer session with the given session ID.
```

Comment:
Lets add docstring for the params in this method. We will also want a CHANGELOG entry for this new public method.